lnd_test.go

```diff
@@ -649,7 +649,7 @@ func testBasicChannelFunding(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("bob didn't report channel: %v", err)
 	}
 
-	// With then channel open, ensure that the amount specified above has
+	// With the channel open, ensure that the amount specified above has
```

Comment:
commit message should describe what is actually fixed. Suggestion

`lnd_test: fix typo in comment`